java/proguard/Portfile

```diff
@@ -21,8 +21,8 @@ homepage        http://proguard.sourceforge.net/
 
 master_sites    sourceforge
 distname        ${name}${version}
-checksums           rmd160  5e4dea659fa4376ce9029f51063b3f3542957786 \
-                    sha256  b7e4e63bd7e232d2b00a962f39da04e504502f0853b69161948de24caf401364
+checksums           rmd160  733100f8442d28d43dadc1cf855e2d1ab1213ced \
+                    sha256  7f64bd1db37c3b2eae69803135e0e59f438b759bd00a5ef18ee7319657916575
```

Comment:
We are now adding the `size` parameter to `checksums`.

Reply:
> We are now adding the `size` parameter to `checksums`.

"We" should tell "port bump" about that, too. ;-)